pkg/core/cloudstorage/provider_impl.go

```diff
@@ -97,7 +97,11 @@ func (p *providerImpl) Sign(scheme string, host string, r *SignRequest) error {
 			Path:   fmt.Sprintf("/_asset/get/%s", assetItem.AssetName),
 		}
 		httpRequest, _ := http.NewRequest("GET", u.String(), nil)
-		httpsigning.Sign(p.secret, httpRequest, now, int(PresignGetExpires.Seconds()))
+		expire := int(PresignGetExpires.Seconds())
+		if assetItem.Expire != 0 {
+			expire = assetItem.Expire
+		}
+		httpsigning.Sign(p.secret, httpRequest, now, expire)
```

Comment:
Should we assign the default `PresignGetExpires.Seconds()` back to ` r.Assets[i].Expire`. if it is empty?

Reply:
I intentionally did not do that because I wanted to keep it as zero value. If the input does not specify `expire`, then `expire` does not appear in the response.

Reply:
Removed `expire` from response